app/models/category.rb

```diff
@@ -0,0 +1,2 @@
+class Category < ApplicationRecord
+end
```

Comment:
What kind of Category. Remember that we will have to deal with Dish categories as well.  Consider the naming standard

Reply:
I see what you mean. Let's change that